docs/fsm.rst

```diff
@@ -131,6 +131,14 @@ secondary will be online to accept writes. When the old primary reaches
 the wait_primary state then the secondary is safe to take offline with
 minimal consequences.
 
+**Prepare_maintenance**
```

Comment:
(unrelated comment)
Wow, I haven't seen `docs/fsm.rst`, good to now. I was actually trying to understand why do we have `demote_timeout` state right now :)